libbeat/processors/add_docker_metadata/add_docker_metadata_test.go

```diff
@@ -11,6 +13,27 @@ import (
 	"github.com/elastic/beats/libbeat/common/docker"
 )
 
+func init() {
+	// Stub out the procfs.
+	processCgroupPaths = func(_ string, pid int) (map[string]string, error) {
+		switch pid {
+		case 1000:
+			return map[string]string{
+				"cpu": "/docker/FABADA",
```

Comment:
Random trivia: Did you know Fabada in Spanish typical meal?, delicious:

![Fabada asturiana](https://upload.wikimedia.org/wikipedia/commons/thumb/7/77/Fabada_en_cazuela_de_barro.jpg/1200px-Fabada_en_cazuela_de_barro.jpg)

Reply:
That looks amazing. I want some.

Reply:
I think I need to order one of these next time I get spanish food.